client/navigation/AuthProvider.tsx

```diff
@@ -13,6 +13,7 @@ export const AuthProvider = ({children}) => {
     const [userToken, setUserToken] = useState(null);
     // response from firestore user collection
     const [userProfile, setUserProfile] = useState(null);
+    const [userLocation, setUserLocation] = useState(null);
```

Comment:
This is a great place to have this! I've been passing it through to components at a lower level but I like it better here where it's accessible globally. Does this mean you can just call  _setUserLocation_ in a component and it will update the value everywhere else?